docs/schema.md

```diff
@@ -400,6 +403,41 @@ properties:
 
 In the example above, manifests can reference to the password as `explicitPassword`, as well as to its base64Encoded value as `explicitPasswordEncoded`.
 
+---
+
+### type: ISTIO_ENABLED
+
+This used to tell the deployer whether Istio is enabled or not. That information is used by manifests to provision special handling to work with Istio.
+
+### Supported values
+- `True`: Istio is not enabled.
+- `False`: Istio is enabled.
+
+If `ISTIO_ENABLED` is not preset, it is assumed Istio is not enabled for the deployment.
+
+---
+
+## istioCompatibility
+
+Nested under `x-google-marketplace`, this indicates whether the application is compatible with Istio.
+
+### Supported values
+- `True`: The app is compatible with Istio.
+- `False`: The app is NOT compatible with Istio.
+
+```yaml
+properties:
+  # Property definitions...
+required:
+  # Required properties...
+x-google-marketplace:
+  istioCompatibility: True # or False
```

Comment:
I predict that there will be additional parameters here. Can we make this into a struct:

```
x-google-marketplace:
  istio:
    isSupported: True
    # Future...
    version: '>0.5'
```

Reply:
That is a very good idea.